src/codemagic/tools/_app_store_connect/actions/publish_action.py

```diff
@@ -55,11 +56,13 @@ class AddBuildToBetaGroupOptions:
 @dataclass
 class SubmitToTestFlightOptions:
     max_build_processing_wait: int
+    expire_previous_builds: Optional[bool] = None
 
 
 @dataclass
 class SubmitToAppStoreOptions:
     max_build_processing_wait: int
+    cancel_previous_submissions: Optional[bool] = None
```

Comment:
As with `SubmitToTestFlightOptions.expire_previous_builds`, this too should have have plain `bool` type:
```suggestion
    cancel_previous_submissions: bool = False
```